src/test/java/za/ac/cput/repository/ScheduleRepositoryTest.java

```diff
@@ -0,0 +1,57 @@
+package za.ac.cput.repository;
+/**
+ * ScheduleRepositoryTest.java
+ * Interface test class
+ * Author: Hlombekazi Mbelu (209024666)
+ * Date: 9 April 2022
+ * */
+import org.junit.jupiter.api.Test;
+import za.ac.cput.entity.Schedule;
+import za.ac.cput.factory.ScheduleFactory;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ScheduleRepositoryTest {
+    public static ScheduleRepository scheduleRepository = ScheduleRepository.getRepository();
+    Schedule schedule = ScheduleFactory.createSchedule(LocalDateTime.of(2022, 4, 7, 2 ,15 ) ,LocalDateTime.now());
+
+    @Test
+    void getRepository() {
+
+    }
+
+    @Test
```

Comment:
Make sure the test methods are done in order. Use the @TestMethodOrder tag ontop of the TestClass.